customersupport/static/list-tickets.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function(){
+    
+    $("#listTicketsButton").click(function(){
+        $.get("api/ticket/customer",{customer_id:$("#customer").val()})
```

Comment:
We should handle the case when the `customer` field is empty. Instead of attempting the API call, maybe show a warning? Or, we could disable the button until the customer field is populated.